### DISCUSSION TITLE
support vue file

### DISCUSSION THREAD

**Discussion by SKT1yang** [oxc-project/oxc#4736] [Q&A] [answered]
Does oxc support. vue files like. js and. ts? I am currently working on a small tool that handles file types including vue files, such as parsing vue into ast, modifying ast, and regenerating vue code. Or if there are any other methods or programs that I don't know about plugins, please let me know. Thank you

**Comment by SKT1yang** [comment]
I found the loader for Vue in oxc_enter ->partial-loader later, but after looking at the source code, it seems to be extracting the script part. However, I want to parse and modify`<template></template>`

**Reply by Boshen** [reply, marked as answer]
Unfortunately we don't have anything for vue :-(
